Adding overview updates from Jim and Ashley (#28)
Ammalgam-Protocol/docs: docs/overview.md

  any pair, leading to what is termed "Impermanent Gain." This concept is also
  known as "borrowing liquidity" or "positive gamma." 

### Capital Efficiency

Market making using uniswap V2 is considered inefficient because most of the
assets are not being used.[^unused_assets] The algorithm to quote swaps has to
support price ranges from 0 to infinity so most of the deposited liquidity is
not supporting trading, but actually sitting idle. Uniswap V3 tried to solve
this problem by introducing concentrated liquidity, which increased yields for
market makers, but also magnified impermanent loss (e.g., increased the rate at
which a market maker's position loses value as price changes). The V3 solution
was, in fact, solving a problem by leveraging the underlying inefficiency -
adding yield by adding risk. Many passive and retail investors are unaware of
the additional risk they are taking with concentrated liquidity.

[^unused_assets]: link?

Ammalgam addresses the issue by first leveraging idle assets in V2 to increase
fees for LPs. If this enhanced capital efficiency does not fully meet trader’s
needs, they have the option to apply additional leverage for further yield
increases. 

> In the tweet below, Hayden suggests LP’s diversify their assets between V3 and

Reviewer: ```suggestion
In the tweet below, Hayden suggests LPs diversify their assets between V3 and
```

lending protocols to mitigate the heightened risks associated with V3 market
making. The DLEX addresses this issue with a single, innovative primitive.
Additionally, it utilizes the unused assets in V2 to generate additional yield
for LPs.

<Tweet id="1452832342788169732" />

### Utility (δ, γ)

The DLEX is far more than a platform for merely lending and trading assets
within a single protocol. Integrating lending and trading as native features
unleashes an array of functionality and trading strategies, surpassing the
capabilities of traditional DEXs or Lending Protocols , whether used
independently or in combination.

This new utility stems from two concepts familiar to options traders which,
Ammalgam now simplifies for retail traders: delta and gamma.

* Delta essentially represents the lending or borrowing of an asset, such as X or
Y on a DLEX. For example, X is USDC and Y is Bitcoin, one can short Bitcoin on a
DLEX by borrowing Bitcoin and selling it for USDC. 
* Gamma refers to exposure in market making, the flip side of the X * Y =K

> formula. WIth gamma on a DLEX, one can borrow K, essentially inverting

Reviewer: ```suggestion
formula. With gamma on a DLEX, one can borrow K, essentially inverting
```

traditional market making. Instead of supplying X and Y, which creates inherent
exposure to impermanent loss, borrowing K flips this exposure to ‘impermanent
gain’.

Delta allows strategies by manipulating borrowed or lent assets. These range
from traditional market making to leveraged market making (akin to concentrated
liquidity) to short market making to delta neutral market making. 

For instance, in the current DeFi space, establishing a delta neutral market
making position requires interacting with multiple protocols, often requiring
advanced technical skills like coding flash loans. While DeFi’s composability
enables interaction with various protocols for a delta neutral position, each
protocol extracts fees. Ammalgam obviates the need for technical skills and

> eliminates the extraneous fees from each protocol,thus recomposing DeFi.

Reviewer: ```suggestion
eliminates the extraneous fees from each protocol, thus recomposing DeFi.
```

* Straddles
* Calls or Puts
* Perpetuals

#### Payout charts

Traders have historically relied on payout charts to determine the value of a
position in relation to price movement. Ammalgam offers users instant access to
assessing the potential payout of a position concerning price fluctuations.

#### Heat map

Many correctly argue that passive LPs are adversely affected by V3, often due to
the amplified effect of impermanent loss from the inherent leverage in
concentrated liquidity. As payout charts remain static over time, these risks
may not be immediately apparent to many retail traders. Ammalgam’s UI introduces
a novel visualization tool that displays the impact of price fluctuations and
the fees earned or incurred by LPs throughout the anticipated duration of their
position. This tool clearly delineates the expected positive (green) or negative
(red) portfolio values in relation to price changes at a future point in time.

![Heat map](/charts/Delta_Neutral_Payout_Heat_Map.png)

### Autonomy

Ammalgam operates independently of oracles and relies solely on itself, except
for interactions with ERC-20 contracts. This autonomy positions Ammalgam to be a

> pioneering platform in both lending and trading for upcoming, major, on chain

Reviewer: ```suggestion
pioneering platform in both lending and trading for upcoming, major, on-chain
```

Author: I actually think this would be correct

```suggestion
pioneering platform in both lending and trading for upcoming, major, onchain
```

airdrops.

This is achieved without relying on assumptions of arbitrage or external trade
liquidity and focused instead on ensuring safety. When initiating borrowing, the
protocol evaluates the available liquidity that could be utilized for
liquidating the proposed debt, as well as existing debts within a similar range.

> This ensures that the new debt can be liquidated using i available internal

Reviewer: what does `i` mean here?

Author: I think it should be 
```suggestion
This ensures that the new debt can be liquidated using an available internal
```
